### ISSUE TITLE
Data Location for Replication

### ISSUE THREAD

**Issue by danieled-csail** [brianhie/geosketch#16]
I tried to install geosketch and replicate the mouse visualziation data as in the readme.

After some issues with packages and installing (which ill not get into too) i ran and got this error
Could not find: data/mouse_brain/dropviz/Cerebellum_ALT

This is DESPITE the fact I installed and extracted the data into the home directory of the git clone. 

Where is the data supposed to be so the code knows where to look for it so bin/mouse_brain_visualize.py
will work properly.

**Comment by brianhie** [comment]
Thanks for reaching out! When you untar the data.tar.gz, can you see the mouse brain data in a subdirectory?

**Comment by danieled-csail** [comment]
Yep. I have all the files in the proper subdirectory identical to how mouse_brain_visaulize.py asks for it.
Its just when i run it it statess file not found

**Comment by danieled-csail** [comment]
Here my current directory tree [most files abstracted away for clearity]
Geosketch
-bin
--mouse_brain_visualize.py.py
-data
--mouse_brain
---zeisel
---dropviz
----Cerebellum_ALT

[other files abstracted awayw for clarity] 

Not sure why it cant find it.
